### PR TITLE
Add reference to Wagner's generalized birthday attack and its subexponential runtime

### LINKED ISSUE #93: Don't overlook Wagner's generalized birthday attack
From Ian's review:

> "Signers can enforce concurrent sessions, though the limit (approximately 256) for reasonably secure elliptic curve groups is small enough to make large-scale signature generation prohibitive.": this sentence makes it sound as though, if you keep concurrency to less than 256, you're safe.  That's not (as I understand it) the case.  The paper shows a very fast attack for 256 using its new [PolytimeROS] attack, but the original subexponential [Wagner02] attack works for much smaller levels of parallelism: for 15 parallel sessions, the attack runs in ~2^55 time (says [FPS20]), which is much lower than the desired security parameter.  Even Clause Blind Schnorr just multiplies the complexity by 2^15 (for 15 parallel sessions), so ~2^70, which is still a far cry from the desired 2^128 security level.

It's true that Wagner's generalized attack applies, so we should try and rewrite this bit to be more clear.

### PR DESCRIPTION
**Pull request by chris-wood**
Closes #93.

### CODE REVIEW

**Comment by jedisct1** [comment]
Maybe remove "though the limit (approximately 256) for  reasonably secure elliptic curve groups is small enough to make large-scale signature generation prohibitive"?

We are talking about two different things, but the juxtaposition can be confusing.

**Comment by chris-wood** [comment]
@jedisct1 good suggestion :-) The new text should be much simpler and more concise.

**Review by jedisct1 — APPROVED** [review]

File: draft-irtf-cfrg-rsa-blind-signatures.md [hunk 1]
```diff
@@ -459,10 +459,11 @@ this specification, despite the shortcomings above.
 signature scheme over elliptic curve groups. Although simple, the hardness problem upon
 which this is based -- Random inhomogeneities in a Overdetermined Solvable system of linear
 equations, or ROS -- can be broken in polynomial time when a small number of concurrent
-signing sessions are invoked {{PolytimeROS}}. This can lead to signature forgeries in practice.
-Signers can enforce a limit on concurrent sessions, though the limit (approximately 256) for
-reasonably secure elliptic curve groups is small enough to make large-scale signature generation
-prohibitive. In contrast, the variant in this specification has no such concurrency limit.
+signing sessions are invoked {{PolytimeROS}}, leading to signature forgeries. However, even
```
[the remark this answers is not in the document]
Comment (chris-wood) [reply draft-irtf-cfrg-rsa-blind-signatures.md:462]: Nevermind! I re-read it and I agree. Dropping. :-)

[the remark this answers is not in the document]
Comment (chris-wood) [reply draft-irtf-cfrg-rsa-blind-signatures.md:462]:
```suggestion
signing sessions are invoked {{PolytimeROS}}, leading to signature forgeries. Even
```

**Comment by FredericJacobs** [comment]
LGTM

**Pull request merged**